client/js/App.vue

```diff
@@ -1,22 +1,16 @@
+<script setup>
+import NavigationDrawer from './components/NavigationDrawer.vue';
```

Comment:
I converted this to use `<script setup>` since that's going to be the default in Vue 3 but I'm curious on your opinion here. I don't know enough about Vuetify to know if it matters that `navigation-drawer` becomes `NavigationDrawer`. I prefer the capitalized multi-word form, and it just happens to work with the way you import in script setup.

Thoughts?